src/librustdoc/doctest.rs

```diff
@@ -1194,6 +1194,13 @@ impl Tester for Collector {
     }
 }
 
+#[cfg(test)] // used in tests
+impl Tester for Vec<usize> {
+    fn add_test(&mut self, _test: String, _config: LangString, line: usize) {
+        self.push(line);
+    }
+}
+
```

Comment:
@GuillaumeGomez (since I can't cc rustdoc)

Is this change acceptable for rustdoc ?

Reply:
What does it do exactly? (A comment in the code is always very welcome :wink: )

Reply:
This `impl` was previously defined here:
https://github.com/rust-lang/rust/blob/dc9acd27965e752fb770e55218b69d896a2e05af/src/librustdoc/html/markdown/tests.rs#L467-L474

But putting an `impl` in a function doesn't limit it's scope, so I just moved it were it makes more sense, and made it conditional on `#[cfg(test)]`.

Reply:
Oh I see. Then no problem. :+1: